pom.xml

```diff
@@ -85,6 +85,19 @@
       <scope>test</scope>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.apache.axis</groupId>
+      <artifactId>axis</artifactId>
+      <version>1.4</version>
+      <type>jar</type>
```

Comment:
I see below you're using the `LRUMap`, but maybe it's a bit misleading to keep `axis` as a dependency isn't it? As we're going away from SOAP to REST here right?